frontend/src/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { createRoot } from "react-dom/client";
+import "./index.css";
+
+import App from "./App";
+import registerServiceWorker from "./registerServiceWorker";
+import schedule from "./data/schedule.json";
+import emptySchedule from "./data/empty_schedule.json";
+
+const rootElement = document.getElementById("root")!;
+rootElement.className = "container-fluid";
+
+const root = createRoot(rootElement);
+root.render(<App schedule={schedule} emptySchedule={emptySchedule} />);
```

Comment:
import and use `schedule` and `emptySchedule` json files directly in App.tsx. I see no point in passing them as props here.